app/controllers/application_controller.rb

```diff
@@ -1,2 +1,8 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+    def configure_permitted_parameters
```

Comment:
we don't add extra spaces for `private` or `protected` methods, also let's not permit these params in the application controller rather in the registration controller for devise.